src/bind.js

```diff
@@ -18,8 +18,11 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...factoryValues) {
+
+  return (...deviceValues) => {
```

Comment:
write it in one line
```suggestion
  return (...deviceValues) => callback(...factoryValues, ...deviceValues);
```